crates/tabby-scheduler/src/lib.rs

```diff
@@ -20,6 +20,14 @@ use tracing::{debug, info};
 
 use crate::doc::SourceDocument;
 
+pub fn format_web_source(url: &str) -> String {
+    format!("website:{url}")
+}
+
+pub fn format_issue_source(provider_id: &str, repository_id: &str) -> String {
+    format!("issue:{provider_id}:{repository_id}")
+}
```

Comment:
this should be in `tabby-webserver`, as scheduler has 0 knowledge of what `issue` / `provider` / `repository` is.